build.gradle

```diff
@@ -17,11 +17,14 @@ apply plugin: 'org.shipkit.gradle-plugin'
 
 apply from: "$rootDir/gradle/upgrade-downstream.gradle"
 
+allprojects {
+    group = 'org.shipkit'
+}
+
 subprojects {
```

Comment:
Perhaps it's simpler to just change "subprojects" -> "allprojects" so that we just have one common section for all projects.